lib/resources/sessions.js

```diff
@@ -24,17 +24,27 @@ module.exports = (service, endpoint) => {
     return Users.getByEmail(email)
       .then(getOrReject(Problem.user.authenticationFailed()))
       .then((user) => bcrypt.verify(password, user.password)
-        .then((verified) => ((verified !== true)
-          ? Problem.user.authenticationFailed()
-          : Sessions.create(user.actor)
```

Comment:
I thought about using `Promise.all()` within the ternary, but that seemed less clear to me. Instead, I started using `rejectIf()`.